NAMESPACE

```diff
@@ -36,5 +36,6 @@ export(rmarkdown_format)
 export(run)
 export(slidy_presentation)
 export(tufte_handout)
+export(tuftish_ebook)
```

Comment:
I don't like the "tuftish" designation nor the "ebook" designation. Is the book not intended to be printed? Why not tufte_book?